DESCRIPTION

```diff
@@ -1,8 +1,17 @@
 Package: pharmaverse
 Title: Navigate 'Pharmaverse'
-Version: 0.0.2
+Version: 0.0.3
 Authors@R: c(
-    person("Ari", "Siggaard Knoph", email = "aikp@novonordisk.com", role = c("aut", "cre")),
+    person("Ari", "Siggaard Knoph", 
+           email = "aikp@novonordisk.com", 
+           role = c("aut", "cre")),
+    person("Ross", "Farrugia", 
+           email = "ross.farrugia@roche.com", 
+           role = c("aut")),
+    person("Mike", "Stackhouse", 
+           email = "mike.stackhouse@atorusresearch.com", 
+           role = c("aut"), 
+           comment = c(ORCID = "0000-0001-6030-723X")),
```

Comment:
what's this out of interest? first time seeing it used

Reply:
Is it the ORCID? It is like a [unique researcher ID](https://orcid.org/).
This is also used in relation to package/software development and the [r-pkgs.org book](https://r-pkgs.org/) has some (sparse) info on it [here](https://r-pkgs.org/description.html#sec-description-authors-at-r).

Reply:
cool! learn something new every day